infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb

```diff
@@ -17,6 +17,8 @@
 
 FROM influxdb:1.7.3
 
+RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
+
```

Comment:
Might as well just update to [`influxdb:2.7.0`](https://hub.docker.com/layers/library/influxdb/2.7.0/images/sha256-32b2c4d5e6fc96da0f6eb325aacc635964d44844cac95bbdee07e620afdf27e4?context=explore), which uses Bullseye

Reply:
I've tried to update to version 2.7.0, and it breaks some of the js for the ui in tp.

Reply:
Ah okay. Does `influxdb:1.8.10` work? If not, we should at least be able to upgrade to `influxdb:1.7.11`.

Reply:
>> I've tried to update to version 2.7.0, and it breaks some of the js for the ui in tp.
>
> Ah okay. Does `influxdb:1.8.10` work? If not, we should at least be able to upgrade to `influxdb:1.7.11`.

`influxdb:1.8.10` uses Bullseye, but `influxdb:1.7.11` still uses Stretch. So if `influxdb:1.8.10` does not work, we'll still need your `sources.list` fix.